www/gatsby-config.js

```diff
@@ -2,6 +2,32 @@ require(`dotenv`).config({
   path: `.env.${process.env.NODE_ENV}`,
 })
 
+const GA = {
+  identifier: `UA-93349937-5`,
```

Comment:
suggest set this also as `process.env`

Reply:
thanks but there is no real benefit to add it as an env var.